LICENSE.txt

```diff
@@ -15,7 +15,7 @@ Additional Use Grant:
     Connecting to a Play Framework websocket and/or Play Framework
     request/response bodies for server and play-ws client.
 
-Change Date:          TBD ( 3 years after 10.5.0 release )
+Change Date:          2026-02-21 ( 3 years after 10.5.0 release )
```

Comment:
I think the original instruction was to just have the date here, see https://github.com/akka/akka/commit/ca7052179650aec65ef9f5e73276f79be6542263

Reply:
👍